src/accessibility/WCAG-2.2/index.md

```diff
@@ -0,0 +1,393 @@
+      },
+      {
+        html: 'Redundant entry'
+      }
+    ],
+    [
+      {
+        html: '<a href="/patterns/passwords/">Passwords</a>'
+      },
+      {
+        html: 'Accessible authentication<br>Consistent help<br>Target size (minimum)'
+      }
+    ],
+    [
+      {
+        html: '<a href="/patterns/payment-card-details/">Payment card details</a>'
+      },
+      {
+        html: 'Redundant entry'
+      }
+    ]
+  ]
+}) }}
+
+{{ govukTable({
+  caption: "Patterns to help users do things",
+  captionClasses: "govuk-table__caption--m",
+  head: [
+    {
+      text: "Help users to...",
+      classes: "govuk-!-width-one-half"
+    },
+    {
+      text: "Relevant WCAG 2.2 criteria",
+      classes: "govuk-!-width-one-half"
+    }
+  ],
+  rows: [
+    [
+      {
+        html: '<a href="/patterns/check-a-service-is-suitable/">Check a service is suitable</a>'
+      },
+      {
+        html: 'Redundant entry'
+      }
+    ],
+    [
+      {
+        html: '<a href="/patterns/check-answers/">Check answers</a>'
+      },
+      {
+        html: 'Redundant entry'
+      }
+    ],
+    [
+      {
+        html: '<a href="/patterns/complete-multiple-tasks/">Complete multiple tasks</a>'
+      },
+      {
+        html: 'Dragging movements<br>Redundant entry'
+      }
+    ],
+    [
+      {
+        html: '<a href="/patterns/confirm-a-phone-number/">Confirm a phone number</a>'
+      },
+      {
+        html: 'Consistent help'
+      }
+    ],
+    [
+      {
+        html: '<a href="/patterns/confirm-an-email-address/">Confirm an email address</a>'
+      },
+      {
+        html: 'Accessible authentication'
+      }
+    ],
+    [
+      {
+        html: '<a href="/patterns/contact-a-department-or-service-team/">Contact a department or service team</a>'
+      },
+      {
+        html: 'Consistent help'
+      }
+    ],
+    [
+      {
+        html: '<a href="/patterns/create-accounts/">Create accounts</a>'
+      },
+      {
+        html: 'Accessible authentication<br>Redundant entry'
+      }
+    ],
+    [
+      {
+        html: '<a href="/patterns/start-using-a-service/">Start using a service</a>'
+      },
+      {
+        html: 'Focus appearance'
+      }
+    ],
+    [
+      {
+        html: '<a href="/patterns/validation/">Recover from validation errors</a>'
+      },
+      {
+        html: 'Redundant entry'
+      }
+    ]
+  ]
+}) }}
+
+{{ govukTable({
+  caption: "Page patterns",
+  captionClasses: "govuk-table__caption--m",
+  head: [
+    {
+      text: "Page",
+      classes: "govuk-!-width-one-half"
+    },
+    {
+      text: "Relevant WCAG 2.2 criteria",
+      classes: "govuk-!-width-one-half"
+    }
+  ],
+  rows: [
+    [
+      {
+        html: '<a href="/patterns/confirmation-pages/">Confirmation pages</a>'
+      },
+      {
+        html: 'Dragging movements<br>Focus appearance<br>Redundant entry'
+      }
+    ],
```

Comment:
Remove highlighting for Focus appearance message , as it's not WCAG 2.2 AA
Remove message for Redundant entry for Confirmation page

```suggestion
    [
      {
        html: '<a href="/patterns/confirmation-pages/">Confirmation pages</a>'
      },
      {
        html: 'Dragging movements'
      }
    ],
```